### ISSUE TITLE
Cannot navigate to folder

### ISSUE THREAD

**Issue by nasli** [owncloud/ios#419]
issue by @ markusphi,
26 de jul. de 2015 14:27 GMT+2

8.0.4 Server version
IPAD 4 MD522/FD Device version
3.4.4 ownCloud app version

Is this a new installation or an update?
Update Own Cloud APP

Do you have more than 1 account registered?
No

Is it possible to download a file? There are 2 issues reported here, in general basis we'd rather 1 report per issue to have a better understanding.

Yes, it ist possible to Download a file from the / folder
but "tab" a Subfolder changes to the Subfolder and "drop back" to /

Additional the App dont store Username / Password

App Version 3.4.1 work without this Problems
Server version
Device version
ownCloud app version
Is this a new installation or an update?
Do you have more than 1 account registered?
Is it possible to download a file? There are 2 issues reported here, in general basis we'd rather 1 report per issue to have a better understanding.
Sorry for my bad english
German: 
Beim tippen eines Unterordner wechselt die Anzeige in den Unterordner, verhält sich dann aber als ob man wieder zurück angewählt hat und landet wieder im / . Außerdem speichert die 3.4.4. App kein Username und Passwort.
Mit der App Version 3.4.1 davor geht es am selben Server einwandfrei.

**Comment by nasli** [comment]
@markusphi, we cannot reproduce it here.
Which iOS version are you using?
What folder structure you have? is it too long? some special characters?
are you using a redirected server?

**Comment by pbek** [comment]
I have the same issue!
OC Server: 8.1.1 with apache & ssl
iOS Client: 3.4.4

It always worked, then maybe after the upgrade to 8.1 or 8.1.1 or an update of the iOS app (I don't know because I didn't use the iOS meanwhile) I wasn't logged in when I opened the app.

So I entered my server & user information, logged in and saw my folders.
After tapping one folder I got the a sliding animation but the root folder got viewed again and the app crashed.
After reopening the app I was logged out again.
I could restart the procedure with the same effect.

UPDATE:
After reinstalling the app the problem was gone. :)
